src/Sql/Sql/ChangeLog.md

```diff
@@ -21,6 +21,12 @@
 * Added cmdlet `New-AzSqlServerTrustGroup`
 * Added cmdlet `Remove-AzSqlServerTrustGroup`
 * Added cmdlet `Get-AzSqlServerTrustGroup`
+* Added cmdlet `Copy-AzSqlDatabaseLongTermRetentionBackup`
+    - Copy LTR backups to different databases
```

Comment:
Copy LTR backups to different server (I think we don't allow same server copy, please confirm with Tilak)

Reply:
We do not allow same server, that is correct